packages/ra-ui-materialui/src/button/UpdateButton.stories.tsx

```diff
@@ -91,23 +95,33 @@ const PostList = () => (
 );
 
 export const InsideAList = () => (
-    <AdminContext dataProvider={getDataProvider()} i18nProvider={i18nProvider}>
-        <AdminUI>
-            <Resource name="posts" list={<PostList />} show={<PostShow />} />
-        </AdminUI>
-    </AdminContext>
+    <TestMemoryRouter>
```

Comment:
**Question**: this test didn't use a custom history before. Did it break if you didn't wrap it with TestMemoryRouter (which would be a sign of BC break), or did you just wrap it for consistency and isolation with the other tests?

Reply:
Yes it is for isolation with the other tests. The story works fine without it but you may land in to a non existing route if you are coming from another story (also not using TestMemoryRouter)